features/step_definitions/auth_steps.rb

```diff
@@ -0,0 +1,32 @@
+Given(/^I have an account on the site$/) do
+  @user = User.last
+end
+
+Given(/^My account is confirmed$/) do
+  @user = User.where("confirmed_at", true).last
```

Comment:
Alternative: Just DRY up the user creation steps and re-run them to click the forms and things.